release-02/code/control.rb

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+
+if ARGV[0] == "init"
+    if File.exist?('/tmp/minitwit.db')
+        puts "Database already exists."
+        exit 1
+    end
+    puts "Putting a database to /tmp/minitwit.db..."
+    system("python -c from minitwit import init_db;init_db()")
```

Comment:
Why are you calling the Python file here. Is it just because you didn't know how the Ruby server will be started?

Reply:
Well I would say that I foolishly copy pasted from the old control.sh. But I don't see what this should become. If you know would you please tell me ? otherwise I'll just spend time looking at it